core/server/worker/src/main/java/alluxio/worker/block/DefaultBlockWorker.java

```diff
@@ -355,6 +358,38 @@ public void removeBlock(long sessionId, long blockId)
     mBlockStore.removeBlock(sessionId, blockId);
   }
 
+  public void freeWorker() throws IOException {
+    IOException ioe = null;
+    FileUtils.deletePathRecursively(Configuration.global().get(PropertyKey.WORKER_PAGE_STORE_DIRS).toString());
```

Comment:
Switch between page store and tiered store by checking `WORKER_BLOCK_STORE_TYPE`

Reply:
Fixed.